analyzers/analyzers.fsproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.Build.NoTargets/1.0.80"> <!-- This is not a project we want to build. -->
+
+  <PropertyGroup>
+    <IsPackable>false</IsPackable>
```

Comment:
I'm a bit hazy on this area of msbuild, but I think `<IsPublishable>false</IsPublishable>` might be another one?

Reply:
Yes, good idea, thanks!